parser_library/src/context/hlasm_context.cpp

```diff
@@ -263,7 +229,7 @@ void hlasm_context::add_scoped_system_variables(system_variable_map& sysvars, si
     } view { *this, skip_last };
 
     sysvars.insert(create_system_variable(id_index("SYSM_SEV"),
-        create_dynamic_var([view]() { return left_pad(std::to_string(view.top().mnote_last_max), 3, '0'); }),
```

Comment:
I haven't checked the rest of the code but do we still need `left_pad` and co.?

Reply:
I don't think we do... that's why I deleted it.